src/importexport/musicxml/imusicxmlconfiguration.h

```diff
@@ -32,27 +32,27 @@ class IMusicXmlConfiguration : MODULE_EXPORT_INTERFACE
 public:
     virtual ~IMusicXmlConfiguration() = default;
 
-    virtual bool musicxmlImportBreaks() const = 0;
-    virtual void setMusicxmlImportBreaks(bool value) = 0;
+    virtual bool musicXmlImportBreaks() const = 0;
```

Comment:
Maybe we can just drop these `musicXml` prefixes here; it's clear enough that these things are about MusicXML :)